### PR TITLE
bug: fix validations for xml #1494

### PR DESCRIPTION
**Pull request by javiersvg**

### CODE REVIEW

**Review by marcingrzejszczak — APPROVED** [review]
One small change should be made in the test but other than that looks good to me!

File: spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/util/xml/XmlToXPathsConverterSpec.groovy [hunk 1]
```diff
@@ -39,6 +45,18 @@ class XmlToXPathsConverterSpec extends Specification {
 		e.message.contains('Prefix must resolve to a namespace: ns1')
 	}
 
+	@Unroll
+	def "should generate [#expectrdValue] for xPath [#value]"() {
```
[the remark this answers is not in the document]
Comment (javiersvg) [reply spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/util/xml/XmlToXPathsConverterSpec.groovy]: Fixed

**Comment by javiersvg** [comment]
One of the tests is failing. I will fix that and let you know when it is ready.

**Comment by javiersvg** [comment]
Test fixed. PR ready to merge.

**Comment by marcingrzejszczak** [comment]
Great work @javiersvg , thanks a lot!

**Pull request merged**